gdx/src/com/badlogic/gdx/math/noise/Noise.java

```diff
@@ -0,0 +1,16 @@
+
+package com.badlogic.gdx.math.noise;
+
+import com.badlogic.gdx.math.Vector3;
+
+public interface Noise {
```

Comment:
The noise interface would be nice but should probably have at least bare-minimum docs for the interface, if not the 5 methods.